cmd/bacalhau/docker_run.go

```diff
@@ -360,15 +370,16 @@ func CreateJob(ctx context.Context, cmdArgs []string, odr *DockerRunOptions) (*m
 				odr.InputUrls = append(odr.InputUrls, i)
 			case "ipfs":
 				odr.InputVolumes = append(odr.InputVolumes, fmt.Sprintf("%s:/inputs", scheme.Host))
+			case "git":
+				odr.InputUrls = append(odr.InputRepos, fmt.Sprintf("https://%s", scheme.Host+scheme.Path))
```

Comment:
What about git repositories that are accessed only over insecure HTTP, or via SSH?

Reply:
Added HTTP support

IDK about SSH I think it may work